test/uk/gov/hmrc/merchandiseinbaggagefrontend/connectors/CurrencyConversionConnectorSpec.scala

```diff
@@ -48,4 +48,19 @@ class CurrencyConversionConnectorSpec extends BaseSpecWithApplication with BaseS
     }
   }
 
+  "get conversion rate for a given currency code" in new CurrencyConversionConnector {
+    override val httpClient: HttpClient = injector.instanceOf[HttpClient]
+
+    override lazy val currencyConversionBaseUrl =
```

Comment:
maybe use a TestCurrencyConversionConnector with the testUrl to avoid the repetition?

Reply:
Not sure if it's necessary given that this will eventually be moved to the backend codebase, so can be implemented there. I don't think the repetition is a big deal personally...

Reply:
I'd have done it but don't feel that strongly!